docs/modules/ROOT/pages/reference/extensions/opentelemetry.adoc

```diff
@@ -56,7 +56,7 @@ In order to send the captured traces to a tracing system, you need to configure
 quarkus.application.name=my-camel-application
 
 # OTLP exporter endpoint
-quarkus.opentelemetry.tracer.exporter.otlp.endpoint=http://localhost:55680
+quarkus.opentelemetry.tracer.exporter.otlp.endpoint=http://localhost:4317
```

Comment:
4317 is some sort of an opentelemetry default?

Reply:
default for OTLP exporter. It's either 4317 or 4318, but the one used in Quarkus doc is 4317 https://quarkus.io/guides/opentelemetry#create-the-configuration